source/includes/_introduction.md.erb

```diff
@@ -32,7 +32,7 @@ The <%= config.apiName %> uses standard HTTP [verbs](http://www.w3.org/Protocols
 To quickly experiment with the API, you can use Postman. You'll need to set the `apiHost` and `apiKey` environment variables.
 
 <div class="postman">
-  <div class="postman-run-button" data-postman-action="collection/import" data-postman-var-1="e238fb8aba0a35436dd3"></div>
+  <div class="postman-run-button" data-postman-action="collection/import" data-postman-var-1="75650d1749860d46aaa0"></div>
```

Comment:
Where is this pointing to?

Reply:
I think a hosted static copy of it by postman, all you have to do is run the button and it'll download you a local copy. Then you can modify and share it, then change the id to the new version of it.